test/src/k-provider/ovp/provider-parser.spec.js

```diff
@@ -35,6 +39,46 @@ describe('provider parser', function() {
       (adaptiveSource === null).should.be.true;
     });
   });
+  describe('_parseProgressiveSource', () => {
+    it('should return a valid progressive sources when getting separate http/s', () => {
+      const context = new playbackContext({});
+      context.flavorAssets = kalturaProgressiveSourceFlavorAssets;
+      const progressiveSource = OVPProviderParser._getParsedSources(
+        [kalturaProgressiveSourceNotSecured, kalturaProgressiveSourceSecured],
+        'myKS',
+        1234,
+        1234,
+        {
+          id: '1_938734'
+        },
+        context
+      );
+      progressiveSource.should.exist;
+      progressiveSource.progressive[0].id.should.equal('0_5407xm9j19951,url');
+      progressiveSource.progressive[0].url.should.equal(
+        'https://cdnapisec.kaltura.com/p/1234/sp/123400/playManifest/entryId/1_938734/protocol/https/format/url/flavorIds/0_5407xm9j/ks/myKS/a.mp4?uiConfId=1234'
+      );
```

Comment:
i think the test in line 57 is enough but if you wish to check also the url please use the data from the MediaConfig and not put that hard coded as someone might want to change it some day in the media.js and this test will fail...